From c1c0358a1c56ce2f8772757868434b968d02fc30 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Tue, 11 Oct 2016 15:47:54 +0800
Subject: [PATCH] arm: dts: add no-relinquish-port property for rk3288-miniarm

This adds support no relinquishing port from ehci to ohci.

Change-Id: I153a85df7407b8e546e75018d71e3763c8f41a10
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-miniarm.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3288-miniarm.dts b/arch/arm/boot/dts/rk3288-miniarm.dts
index e1177dd97dcb..e4ecd2036c19 100644
--- a/arch/arm/boot/dts/rk3288-miniarm.dts
+++ b/arch/arm/boot/dts/rk3288-miniarm.dts
@@ -428,6 +428,7 @@
 };
 
 &usb_host0_ehci {
+	no-relinquish-port;
 	status = "okay";
 };
 
-- 
2.35.3

